src/emit/mod.rs

```diff
@@ -5075,10 +5090,10 @@ pub trait TargetRuntime<'a> {
                     }
                     // bits / 3 is a rough over-estimate of how many decimals we need
```

Comment:
You've updated the division, but this comment also needs updating.

Reply:
Fixed